fast_barcode_scanner/android/src/main/kotlin/com/jhoogstraat/fast_barcode_scanner/FastBarcodeScannerPlugin.kt

```diff
@@ -183,7 +216,11 @@ class FastBarcodeScannerPlugin : FlutterPlugin, MethodCallHandler, StreamHandler
             pluginBinding.textureRegistry.createSurfaceTexture(),
             configuration
         ) { barcodes ->
-            detectionEventSink?.success(encode(barcodes))
+            detectionEventSink?.success(
+                mapOf(
+                    "barcodes" to encode(barcodes),
+                )
+            )
```

Comment:
It doesn't seem like changing this from a list to a map is necessary. The upgrade path will be easier if we keep it the same.